### PR TITLE
Fixes #11: Updated record for N. Cyprus

### LINKED ISSUE #11: Northern Cyprus disputed area name and hierarchy funky
For wof_id
85632375

name now:
N. Cyprus
name proposed:
Northern Cyprus

data/856/323/75/85632375.geojson

parent now:
85632437 (Cyprus)
parent proposed:
102191569 (Europe)

And it's only claimed by Cyprus, not also
* the country of Kosovo
* the country of Serbia

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #11. Updated the record for [N. Cyprus](https://whosonfirst.mapzen.com/spelunker/id/85632375/#9/35.3543/33.5921).

* Changed `wof:name` from `N. Cyprus` to `Northern Cyprus`
* Removed the following from the `wof:belongs` and `wof:belongsto` fields:
    * the Country of Kosovo
    * the Country of Serbia

* Exportified to update `geomhash` and `geometry`

### CODE REVIEW

File: data/856/323/75/85632375.geojson [hunk 1]
```diff
@@ -183,42 +183,28 @@
     "wof:belongs":[
         85632437,
         102191581,
-        85633259,
-        85632375,
-        85633755
+        85632375
     ],
     "wof:belongsto":[
         102191581,
```
Comment (nvkelso) [inline data/856/323/75/85632375.geojson]: Like it's neighbour, Northern Cyprus is in Asia: `102191569` (not `81`).

File: data/856/323/75/85632375.geojson [hunk 2]
```diff
@@ -183,42 +183,28 @@
     "wof:belongs":[
         85632437,
         102191581,
```
Comment (nvkelso) [inline data/856/323/75/85632375.geojson]: Should be: `102191569`

File: data/856/323/75/85632375.geojson [hunk 3]
```diff
@@ -183,42 +183,28 @@
...
-            "continent_id":102191581,
-            "country_id":85633755,
-            "disputed_id":85632375
         }
     ],
     "wof:id":85632375,
-    "wof:lastmodified":1452291003,
-    "wof:name":"N. Cyprus",
+    "wof:lastmodified":1463417342,
+    "wof:name":"Northern Cyprus",
```
Comment (nvkelso) [inline data/856/323/75/85632375.geojson]: 👍

**Comment by stepps00** [comment]
Most recent [commit](72ff3dc) replaced all instances of `102191581` (Asia) with `102191569` (Europe). Also updated `wof:lastmodified` field.

File: data/856/323/75/85632375.geojson [hunk 4]
```diff
@@ -182,43 +182,29 @@
...
     },
     "wof:country":"XX",
-    "wof:geomhash":"428cfad5718f2d425475dc814d6bb357",
+    "wof:geomhash":"0d736013b14f15cfb90cf960bc69661c",
     "wof:hierarchy":[
         {
-            "continent_id":102191581,
+            "continent_id":102191569,
             "country_id":85632437,
             "disputed_id":85632375
```
Comment (nvkelso) [inline data/856/323/75/85632375.geojson]: One more change:

The northern part of the island is disputed by Cyprus (the southern part), so we need to specify it is independently itself, and claimed by the south in the hierarchy: 

```
"wof:hierarchy":[
{
              "continent_id":102191569,
              "country_id": 85632375,
              "disputed_id":85632375}, 
{
              "continent_id":102191569,
              "country_id": 85632437,
              "disputed_id":85632375
}
],
```

@thisisaaronland for sanity check on above.

Comment (stepps00) [inline data/856/323/75/85632375.geojson]: Updated `wof:hierarchy` in [most recent commit](ecb0dc9badf094bb8f8448b580c9c1bdf3090d6c).

**Comment by thisisaaronland** [comment]
👍

**Pull request merged**